### PR TITLE
feat: [ADN-47] Create a transfer token summary component

### PR DESCRIPTION
**Pull request by jinoosss**
This component is used on the transfer token summary page.

### CODE REVIEW

File: packages/adena-extension/src/components/transfer/transfer-summary-network-fee/transfer-summary-network-fee.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import { Meta, StoryObj } from '@storybook/react';
+
+export default {
+  title: 'components/transfer/TransferSummaryNetworkFee',
+  component: TransferSummaryNetworkFee,
+} as Meta<typeof TransferSummaryNetworkFee>;
+
+export const Default: StoryObj<TransferSummaryNetworkFeeProps> = {
+  args: {
+    networkFee: ''
```
Comment (luvhaeun) [inline packages/adena-extension/src/components/transfer/transfer-summary-network-fee/transfer-summary-network-fee.stories.tsx:11]: Enter a value that is commonly used by default

**Review by luvhaeun — CHANGES_REQUESTED** [review]
I left one comment.

File: packages/adena-extension/src/components/transfer/transfer-summary-network-fee/transfer-summary-network-fee.stories.tsx (hunk 1 above)
Comment (jinoosss) [reply packages/adena-extension/src/components/transfer/transfer-summary-network-fee/transfer-summary-network-fee.stories.tsx]: Added Storybook default data and changed to use the Balance component.

**Review by luvhaeun — APPROVED** [review]
GOOOOD

**Pull request merged**